### DISCUSSION TITLE
Html video

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#6537] [Help]
why video tag not working

![Capture](https://user-images.githubusercontent.com/106697208/177118502-e2d29b0d-d81e-4bfe-8a32-265098e27aaa.PNG)

**Comment by melohagan** [comment]
You can't access the **window** object in an Embed component.
Try it without the *onended* attribute.

**Comment by rafikApp** [comment]
But it works well with img tag. i think the problem is with video tag
The idea is that I want to navigate to another page as soon as the video ends. I tried with an image (gif) using the setTime function (to execute a script after a delay) but sometimes it executes the script before the end of the animation and sometimes after because of the change in the connection speed between the devices (connection problem)
